libbeat/idxmgmt/idxmgmt.go

```diff
@@ -57,25 +51,40 @@ type Supporter interface {
 
 	// Manager creates a new manager that can be used to execute the required steps
 	// for initializing an index, ILM policies, and write aliases.
-	Manager(client ESClient, assets Asseter) Manager
+	Manager(client ClientHandler, assets Asseter) Manager
 }
 
 // Asseter provides access to beats assets required to load the template.
 type Asseter interface {
 	Fields(name string) []byte
 }
 
-// ESClient defines the minimal interface required for the index manager to
-// prepare an index.
-type ESClient interface {
-	Request(method, path string, pipeline string, params map[string]string, body interface{}) (int, []byte, error)
-	GetVersion() common.Version
-}
-
 // Manager is used to initialize indices, ILM policies, and aliases within the
 // Elastic Stack.
 type Manager interface {
-	Setup(forceTemplate, forcePolicy bool) error
+	Setup(template, ilm LoadMode) error
+}
+
+// LoadMode defines the mode to be used for loading idxmgmt related information.
+// It will be used in combination with idxmgmt configuration settings.
+type LoadMode uint8
+
+//go:generate stringer -linecomment -type LoadMode
```

Comment:
did you run `go generate`? I'm missing the file with the `String` method implementation on LoadMode.
Without comments behind the symbols the `String` method will return "LoadModeEnabled" and so on.

Reply:
yes, I missed to actually commit & push the file